rsrc/schemas/dialog.xsd

```diff
@@ -229,6 +236,7 @@
 				<xs:attributeGroup ref="rect-size"/>
 				<xs:attributeGroup ref="position"/>
 				<xs:attributeGroup ref="font"/>
+				<xs:attribute name="text-side" default="right" type="textSide"/>
```

Comment:
I don't really like this name for it. "alignment" might be a bit better but is also not that great. Maybe "attachment"? Not sure I like that name either… got any other ideas?